8.x/getting-started/contributing.md

```diff
@@ -387,19 +387,19 @@ ___
 
 ## PR review
 
-This section describes how to run, test and review your teammate's pull request in your local system.
+This section describes how to run, test, and review your teammate's pull request using an editor on your local system.
 
 ### Git
 
-`Git`*<sup>[4](#4)</sup>* is a `version control` or `source control` system which saves the source code and remembers the source code `changes` step by step as a stack. This stack is called `branch` and the changes are called `commit` or `revision`.
+`Git`*<sup>[4](#4)</sup>* is a `version control` or `source control` system which enables source code saving and tracking of the `changes` in a step-by-step manner, forming a `stack` or `branch`. Each change in the branch is referred to as a `commit` or `revision`.
 
-If you order to cancel a commit, git creates a new commit which reverts the previous commit changes. So no commit is deleted from the git, but always the new commits are added. Git performs this task by adding and subtracting the lines. For example `+5 -3` in a given commit means that you added 5 lines to the source code and at the same time deleted 3 lines from it. Now if you order to delete this commit, git deletes the 5 new lines and adds the previous 3 lines and creates a new commit.
+In Git, canceling a commit results in the creation of a new commit which reverts the previous commit changes. So no commit is ever deleted; instead, new ones are added. Git performs this task by adding or subtracting the lines. For example `+5 -3` in a given commit indicates that five lines were added to the source code while three were deleted. If you order to delete this commit, Git deletes the five newly added lines and adds back the previous three lines, creating a new commit.
```

Comment:
previous commit changes --> previous changes or previously committed changes.

Reply:
@reyhane-rangamiz : I mean : "the changes of the previous commit".

Reply:
@ShammasAmir  So you should say `previous commit's changes`

Reply:
@reyhane-rangamiz : My screenshot from google translate:
![google-translate](https://user-images.githubusercontent.com/90921939/235606739-07bd0d92-49d1-4017-bac9-6a783bd020a3.png)

Reply:
@ShammasAmir 
"Previous commit changes" refers to the changes that were made in the commit immediately before the current one.
"Previous commit's changes" refers to the changes that were made in the previous commit itself.
So use them based on what you want to convey. if you meant the first one, yes you are correct, and sorry I misunderstood.

Reply:
@reyhane-rangamiz : I mean the first one. Thanks for your description.